sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -3067,22 +3096,29 @@ def test_pretty_derivatives():
 x  + ─────(2⋅x⋅y)\n\
      ∂x ∂y       \
 """
-    assert pretty(expr) in [ascii_str_1, ascii_str_2]
-    assert upretty(expr) in [ucode_str_1, ucode_str_2]
+    ucode_str_3 = \
+"""\
```

Comment:
All lines are the same length (in terms of characters); variable width characters make it look otherwise.